atlp-rwanda/dynamites-ecomm-be
# logging set up

### What does this PR do?
This PR establishes set up logging library and configuration of logging level  and output them into the file called output.log 
### Description of Task to be completed?
 -Set up a logging library
- Configure logging levels and output destinations
### How should this be manually tested?
after  starting  server you can send request  using  postman , swagger ,.. after check in the output.log 
### Run Commands
- for start server : `npm run dev`
### related issue
#12
